instance/instance.go

```diff
@@ -154,13 +176,20 @@ func (s *Store) Update(w http.ResponseWriter, r *http.Request) {
 		var editionDoc *models.Edition
 
 		datasetID := currentInstance.Links.Dataset.ID
-		instance.Links.Job = currentInstance.Links.Job
+		if instance.Links != nil {
+			instance.Links = currentInstance.Links
+		} else {
+			instance.Links = &models.InstanceLinks{
+				Job:     currentInstance.Links.Job,
+				Self:    currentInstance.Links.Self,
+				Spatial: currentInstance.Links.Spatial,
+			}
+		}
```

Comment:
It seems like either case results in the instance.links being replaced entirely - if this is right, could it be simplified to not have a if statement and just use `instance.Links = currentInstance.Links`?

Reply:
See comment above